### PR TITLE
scidoc fix for PR626

### PR DESCRIPTION
**Pull request by mzhangw**
Here is a doxyen fix for PR626

### CODE REVIEW

**Review by ligiabernardet — APPROVED** [review]
Looks good to me. Note that the use of ver="5" in the SDF is compatible with the v5 public release but not with the current master code. In general, the documentation we are putting in master matches v5, and not the master itself. There is an understanding that we will update the documentation before the next public release.

**Comment by climbfuji** [comment]
> Looks good to me. Note that the use of ver="5" in the SDF is compatible with the v5 public release but not with the current master code. In general, the documentation we are putting in master matches v5, and not the master itself. There is an understanding that we will update the documentation before the next public release.

This is a bit of a problem. The version string in the suite definition file no longer corresponds to the version of the CCPP release, but to the format/syntax of the suite definition file. This is what we agreed to with our NCAR colleagues a few months ago and we made the change to `version="1"` with this understanding. So this must be `version="1"`, not `5`.

**Comment by climbfuji** [comment]
The meeting minutes for this discussion are here: https://github.com/NCAR/ccpp-framework/wiki/CCPP-Framework-Meeting-Minutes-2020-08-27

**Review by climbfuji — CHANGES_REQUESTED** [review]
Replace version="5" with version="1" everywhere.

**Comment by ligiabernardet** [comment]
Sure, but the Scientific Documentation that Laurie is proposing to commit to master is all about the v5 release and lags behind what is actually on trunk. I suggest that we go forward with updating the master with innovations from v5, and we can proceed with additional updates updates of documentation afterwards.

**Comment by mzhangw** [comment]
Somehow I just updated as Dom requested.

**Comment by ligiabernardet** [comment]
OK. In this case I believe` lib="ccppphys"` needs to be removed, as it is no longer used in the SDFs.

File: physics/docs/pdftxt/GFSv15p2_suite.txt [hunk 1]
```diff
@@ -26,7 +26,7 @@ The GFS_v15p2 physics suite uses the parameterizations in the following order:
 \code
 <?xml version="1.0" encoding="UTF-8"?>
 
-<suite name="FV3_GFS_v15p2" lib="ccppphys" ver="5">
```
Comment (climbfuji) [inline physics/docs/pdftxt/GFSv15p2_suite.txt:29]: Please replace `lib="ccppphys" ver="1"` with just `version="1"` to be consistent with the new syntax.

File: physics/docs/pdftxt/GFSv16beta_suite.txt [hunk 2]
```diff
@@ -29,7 +29,7 @@ The GFS_v16beta physics suite uses the parameterizations in the following order:
 \code
 <?xml version="1.0" encoding="UTF-8"?>
 
-<suite name="FV3_GFS_v16beta" lib="ccppphys" ver="5">
```
Comment (climbfuji) [inline physics/docs/pdftxt/GFSv16beta_suite.txt:32]: Please replace `lib="ccppphys" ver="1"` with just `version="1"` to be consistent with the new syntax.

File: physics/docs/pdftxt/RRFS_v1alpha_suite.txt [hunk 3]
```diff
@@ -24,7 +24,7 @@ The RRFS_v1alpha physics suite uses the parameterizations in the following order
 \code
 <?xml version="1.0" encoding="UTF-8"?>
 
-<suite name="FV3_RRFS_v1alpha" lib="ccppphys" ver="5">
```
Comment (climbfuji) [inline physics/docs/pdftxt/RRFS_v1alpha_suite.txt:27]: Please replace `lib="ccppphys" ver="1"` with just `version="1"` to be consistent with the new syntax.

**Review by climbfuji — CHANGES_REQUESTED** [review]
Thanks for your changes so far. When release code goes back to the authoritative repositories, it needs to be adjusted to the innovations that have been made in the meantime, and one of them is the agreement on the new SDF syntax. Thus, replacing `lib="ccppphys" ver="5"` (or `lib="ccppphys" ver="1"` in places where you made such changes already) with just `version="1"` is correct.

**Review by climbfuji — APPROVED** [review]

**Comment by mzhangw** [comment]
Issue https://github.com/NCAR/ccpp-physics/issues/581 is solved

**Comment by climbfuji** [comment]
> [NCAR#581](https://github.com/NCAR/ccpp-physics/issues/581) is solved

Thanks @mzhangw . Laurie, please addd to the description of your PR
> Fixes #581
so that the issue gets closed once your code gets merged. Thanks!

**Pull request merged**